lib/omnibus/packagers/bff.rb

```diff
@@ -229,6 +232,9 @@ def create_bff_file
       FileSyncer.glob(File.join(staging_dir, 'tmp/*.bff')).each do |bff|
         copy_file(bff, File.join(Config.package_dir, create_bff_file_name))
       end
+
+      # chown back to original user's uid/gid so cleanup works correctly
+      shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
```

Comment:
@schisamo Unfortunately, doing the manual steps cause the files inside the resulting package to be owned by the existing user when they are installed on a target machine. The chown/sudo 'fix' was actually set up to fix a bug where packages were created with my username and wouldn't install correctly on target machines. Short of patching mkinstallp (which I seriously don't think we should do) I don't see a solution other than sudo.